simpleLooping/test_binaryListToString.py

```diff
@@ -0,0 +1,12 @@
+import unittest
+from binaryListToString import binaryListToString
+
+class TestBinaryListToString(unittest.TestCase):
```

Comment:
Did you check the case where the list is empty? 

It is important because for example “num = 10” and “num = 0” are integer but “num = None” is not.